core/src/test/scala/integration/kafka/api/AdminClientIntegrationTest.scala

```diff
@@ -75,11 +81,12 @@ class AdminClientIntegrationTest extends KafkaServerTestHarness with Logging {
     super.tearDown()
   }
 
-  val brokerCount = 3
-  lazy val serverConfig = new Properties
+  val serverCount = 3
```

Comment:
Is there a reason to change brokerCount to serverCount?  brokerCount seems clearer to me?

Reply:
It's because now the `AdminClientIntegrationTest` class is derived from `IntegrationTestHarness` where we have `serverCount` alredy defined. We should change `serverCount` to `brokerCount` in such base class. It means to change the name for other class derived from `IntegrationTestHarness` (i.e. `MetricsTest` and so on). Wdyt ?

Reply:
@cmccabe wdyt about that ?